lib/ClangImporter/ImportType.cpp

```diff
@@ -2594,7 +2594,7 @@ static ParamDecl *getParameterInfo(ClangImporter::Implementation *impl,
     if (CallExpr *defaultArgExpr = synthesizer.makeDefaultArgument(
             param, swiftParamTy, paramInfo->getParameterNameLoc())) {
       paramInfo->setDefaultArgumentKind(DefaultArgumentKind::Normal);
-      paramInfo->setDefaultExpr(defaultArgExpr, /*isTypeChecked*/ true);
+      paramInfo->setDefaultExpr(defaultArgExpr);
       paramInfo->setDefaultValueStringRepresentation("cxxDefaultArg");
     }
   }
```

Comment:
@xedin @egorzhdan Should I special-case this? Was there a particular reason to fill in semantic info in place rather than having the type-checker do the job?

Reply:
Looks like this is just something that type-checker wouldn't be able to handle at the moment.